### PR TITLE
Change translations to match last design copies

### PR DESCRIPTION
**Pull request by mela124**
Updated some english copies and their translations. Now uses Schibsted account instead of SPiD

### CODE REVIEW

File: ui/src/main/res/values-nb/strings.xml [hunk 1]
```diff
@@ -19,18 +18,17 @@
     <string name="schacc_email_privacy_information">Vi vil aldri dele e-postadressen din uten ditt samtykke.</string>
     <string name="schacc_email_identification_error">Oppgi en gyldig e-postadresse.</string>
     <string name="schacc_email_label">Skriv inn din e-postadresse</string>
-    <string name="schacc_email_hint">fornavn.etternavn@e-post.com</string>
+    <string name="schacc_email_hint"></string>
```
Comment (Astach) [inline ui/src/main/res/values-nb/strings.xml]: is that normal?

File: ui/src/main/res/values-nb/strings.xml [hunk 2]
```diff
@@ -82,21 +80,21 @@
...
     <string name="schacc_welcome_back_title">Velkommen tilbake</string>
     <string name="schacc_register_title">Opprett bruker</string>
     <string name="schacc_ok">OK</string>
+    <string name="schacc_identifier_modifier">Ikke deg?</string>
     <!--Generic error messages-->
     <string name="schacc_generic_exception_error_message_title">Noe gikk galt!</string>
     <string name="schacc_generic_time_out_error_message">Tilkoblingen ble tidsavbrutt. Prøv igjen.</string>
     <string name="schacc_generic_server_error_message">Vi har problemer for øyeblikket. Prøv igjen senere.</string>
-    <string name="schacc_generic_internet_error" tools:ignore="Typos">Kan ikke koble til internett. Sjekk tilkoblingen din.</string>
+    <string name="schacc_generic_internet_error">Kan ikke koble til internett. Sjekk tilkoblingen din.</string>
```
Comment (Astach) [inline ui/src/main/res/values-nb/strings.xml]: tools:ignore="Typos" has to be there

File: build.gradle [hunk 3]
```diff
@@ -17,7 +17,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.1.0-rc01'
```
Comment (Nexcius) [inline build.gradle]: This is a no-go

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
-#Mon Oct 30 12:13:42 CET 2017
+#Tue Apr 03 09:00:47 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (Nexcius) [inline gradle/wrapper/gradle-wrapper.properties]: Remove from change-set

**Pull request merged**